src/clj/fluree/db/query/api.cljc

```diff
@@ -207,11 +208,9 @@
         (dataset db-map defaults)))))
 
 (defn query-connection-fql
-  [conn query]
+  [conn query {:keys [did] :as opts}]
```

Comment:
I think we should pass `did` directly here instead of introducing another open-ended options argument as this isn't a user-facing function

Reply:
Ok! Done in eaecd865cbebedee14a1cb314c4a9c3568964e16